### PR TITLE
replace custom .gif loader w/ font-awesome spinner

### PR DESCRIPTION
**Pull request by shapiromatron**
A long time ago in a galaxy far away when we started this project, there was no webpack or react and just jQuery. We had a custom loading gif that worked great for its time, but have since moved into a different more component-based world. This PR removes the old gif and uses a font-awesome spinner to be consistent with other loaders in HAWC.

Screen Shot 2021-05-27 at 10 54 55 AM

URLs to check in dev db:
- visual: http://127.0.0.1:8000/summary/assessment/100000061/visuals/
- pubmed search: http://127.0.0.1:8000/lit/assessment/100500007/search/outbreak-salmonella-agona-europe/
- reference search: http://127.0.0.1:8000/lit/assessment/100000037/references/search/
- dp: http://127.0.0.1:8000/summary/data-pivot/assessment/100000037/pfbs-candidate-pods-subchronic-and-chronic-rfd/
- dp update: http://127.0.0.1:8000/summary/data-pivot/assessment/100000037/pfbs-candidate-pods-subchronic-and-chronic-rfd/update/

### CODE REVIEW

File: frontend/summary/summaryForms/BaseVisualForm.js [hunk 1]
```diff
@@ -209,15 +208,16 @@ class BaseVisualForm {
 
     setPreviewLoading() {
         var $preview = this.$el.find("#preview"),
-            loading = $('<p class="loader">Loading... <img src="/static/img/loading.gif"></p>');
+            loading = HAWCUtils.loader();
```
Comment (rabstejnek) [inline frontend/summary/summaryForms/BaseVisualForm.js:211]: Should be `HAWCUtils.loading()`

File: frontend/summary/summaryForms/BaseVisualForm.js [hunk 2]
```diff
@@ -209,15 +208,16 @@ class BaseVisualForm {
...
         $preview.html(loading);
     }
 
     addSettingsLoader() {
         var div = $("#settings"),
             loader = div.find("p.loader");
-        if (loader.length === 0)
-            loader = $('<p class="loader">Loading... <img src="/static/img/loading.gif"></p>');
+        if (loader.length === 0) {
+            loader = HAWCUtils.loader();
```
Comment (rabstejnek) [inline frontend/summary/summaryForms/BaseVisualForm.js:219]: Should also be `HAWCUtils.loading()`

**Review by rabstejnek — CHANGES_REQUESTED** [review]
There are two function calls to `loader` instead of `loading`, but with that fixed it looks good to go!

File: frontend/summary/summaryForms/BaseVisualForm.js (hunk 2 above)
Comment (shapiromatron) [reply frontend/summary/summaryForms/BaseVisualForm.js]: thanks, i'm glad you noticed this!

**Review by rabstejnek — APPROVED** [review]

**Pull request merged**